eth/deployer/rollup/rollup.go

```diff
@@ -253,13 +260,32 @@ func DeployConfiguredRollup(c chain.Connection, cfg DeploymentConfig) (*RollupCo
 	}
 
 	// Stage 7
```

Comment:
Let's update or remove `deploy_sequence.png` diagram

Reply:
I think it's worth to update this diagram instead of removing it.